test/functional/rpc_net.py

```diff
@@ -199,6 +201,134 @@ def test_getnettotals(self):
+
+        self.log.info(f"Test full (un-aggregated) output is as expected")
+        stats_full = node0.getnetmsgstats()
+        assert_equal(
+            stats_full,
+            {
+                "recv": {
+                    "not_publicly_routable": {
+                        "manual": {
+                            "addrv2": {"bytes": 63 if self.options.v2transport else 66, "count": 1},
+                            "feefilter": {"bytes": 29 if self.options.v2transport else 32, "count": 1},
+                            "getheaders": {"bytes": 666 if self.options.v2transport else 669, "count": 1},
+                            "headers": {"bytes": 103 if self.options.v2transport else 106, "count": 1},
+                            "ping": {"bytes": 29 if self.options.v2transport else 32, "count": 1},
+                            "pong": {"bytes": 29 if self.options.v2transport else 32, "count": 1},
+                            "sendaddrv2": {"bytes": 33 if self.options.v2transport else 24, "count": 1},
+                            "sendcmpct": {"bytes": 30 if self.options.v2transport else 33, "count": 1},
+                            "sendheaders": {"bytes": 33 if self.options.v2transport else 24, "count": 1},
+                            "verack": {"bytes": 33 if self.options.v2transport else 24, "count": 1},
+                            "version": {"bytes": 147 if self.options.v2transport else 138, "count": 1},
+                            "wtxidrelay": {"bytes": 33 if self.options.v2transport else 24, "count": 1},
+                        }
+                    }
+                },
+                "sent": {
+                    "not_publicly_routable": {
+                        "manual": {
+                            "feefilter": {"bytes": 29 if self.options.v2transport else 32, "count": 1},
+                            "getaddr": {"bytes": 33 if self.options.v2transport else 24, "count": 1},
+                            "getheaders": {"bytes": 666 if self.options.v2transport else 669, "count": 1},
+                            "headers": {"bytes": 103 if self.options.v2transport else 106, "count": 1},
+                            "ping": {"bytes": 29 if self.options.v2transport else 32, "count": 1},
+                            "pong": {"bytes": 29 if self.options.v2transport else 32, "count": 1},
+                            "sendaddrv2": {"bytes": 33 if self.options.v2transport else 24, "count": 1},
+                            "sendcmpct": {"bytes": 30 if self.options.v2transport else 33, "count": 1},
+                            "sendheaders": {"bytes": 33 if self.options.v2transport else 24, "count": 1},
+                            "verack": {"bytes": 33 if self.options.v2transport else 24, "count": 1},
+                            "version": {"bytes": 147 if self.options.v2transport else 138, "count": 1},
+                            "wtxidrelay": {"bytes": 33 if self.options.v2transport else 24, "count": 1},
```

Comment:
for lines 233-262, it might be more efficient to replace all of these `if self.options.v2transport` conditionals with just one (and if v1 is ever removed, doing so would be a cleaner diff)

Reply:
Done